xcell/mappers/mapper_DESY1wl.py

```diff
@@ -250,23 +221,18 @@ def get_nl_coupled(self, mode=None):
             self.nl_coupled = self.nls[mod]
             return self.nl_coupled
 
-        # This will only be computed if self.maps['mod'] is None
-        file_name = \
-            f'DESwlMETACAL_{mod}_w2s2_zbin{self.zbin}_ns{self.nside}.fits.gz'
-        read_lite, fname_lite = self._check_lite_exists(file_name)
-
-        if read_lite:
-            print('Loading w2s2 bin{} map'.format(self.zbin))
-            w2s2 = hp.read_map(fname_lite)
-        else:
+        # This will only be computed if self.nls['mod'] is None
+        def get_w2s2():
```

Comment:
Note that `get_w2s2` was not a method before, so we're not losing any functionality.

The main reason for putting it inside is that otherwise the function would depend on a parameter (`mode`), and the functions we pass `_rerun_read_cycle` cannot depend on anything.

If we want this for debugging we just need to look at the map that gets saved into the rerun file.

Reply:
I think other weak lensing mappers have it outside (as HSC). But this is what was implemented. If we want to homogenize that later, we can do it in a different PR